### PR TITLE
Fix show stats

### PR DESCRIPTION
**Pull request by critical27**
Both BlockBasedTable and PlainTable will failed in some certain case.

1. Both will failed when use HashSkipList as memtable.
2. BlockBasedTable will fail when specify `prefix_same_as_start`, it is ok to remove it since we check it externally.
 
Will dig more later and add some test case.

### CODE REVIEW

File: src/kvstore/RocksEngine.cpp [hunk 1]
```diff
@@ -234,7 +233,6 @@ RocksEngine::rangeWithPrefix(const std::string& start,
                              const std::string& prefix,
                              std::unique_ptr<KVIterator>* storageIter) {
     rocksdb::ReadOptions options;
-    options.prefix_same_as_start = true;
```
Comment (panda-sheep) [inline src/kvstore/RocksEngine.cpp:237]: Good job，thx
Perhaps just removing `prefix_same_as_start` is enough. In this way rocksdb uses the default value of `prefix_same_as_start`.

**Review by liuyu85cn — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Comment by critical27** [comment]
Hold on a sec, do some research.

**Comment by critical27** [comment]
Ready....

**Review by bright-starry-sky — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]

**Pull request merged**